sparql/reports/owl-missing-biolink-term.rq

```diff
@@ -12,7 +12,8 @@ WHERE {
   FILTER NOT EXISTS {
 	?n0 rdf:type ?bl_type .
     ?bl_type rdf:type metatype:ClassDefinition .
-    FILTER(?bl_type != bl:NamedThing)
+    OPTIONAL{ bl:NamedThing rdfs:subclassOf ?entity } .
```

Comment:
Should I add a constraint that `?entity rdf:type blml:types/ClassDefinition`? (Ref: [)](https://api.triplydb.com/s/TAkcLrX_c)